CHANGELOG.md

```diff
@@ -14,7 +14,12 @@ _None_
 
 ### New Features
 
-_None_
+* More variables have been deprecated, while new variables have been added.  
+  [David Jennes](https://github.com/djbe)
+  [#13](https://github.com/SwiftGen/SwiftGenKit/issues/13)
+  * For each string, the `params` variable and it's subvariables (such as `names`, `count`, ...) have been replaced by `parameters`, which is an array of types.
+  * The `strings`, `structuredStrings` and `tableName` have been replaced by `tables`, which is an array of string tables, each with a `name` and a `strings` property.
```

Comment:
Nitpicking but I find it more readable / logical to inverse the order of the first 2 bullet points. First talk top level, then talk about structure for each string.